### ISSUE TITLE
Add global version

### ISSUE THREAD

**Issue by QU35T-code** [GoToolSharing/htb-cli#70]
const Version = "1.0.0"

**Comment by DisplayGFX** [comment]
It might be a good idea to also set the dev version to be the commit hash.

**Comment by QU35T-code** [comment]
Hey @DisplayGFX,

Dev has now a version (commit hash)

```bash
> go run . update
A new update is now available (dev) ! (fd2651baa419e8d022c3d8968a781552d87a923a)
Update with : git pull
```

```bash
> git pull
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Total 4 (delta 2), reused 4 (delta 2), pack-reused 0
Unpacking objects: 100% (4/4), 373 bytes | 373.00 KiB/s, done.
From github.com:GoToolSharing/htb-cli
   fd2651b..f9f4619  dev        -> origin/dev
Updating fd2651b..f9f4619
Fast-forward
 config/config.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
```

```bash
> go run . update
You're up to date (dev) ! (fd2651baa419e8d022c3d8968a781552d87a923a)
```
